### ISSUE TITLE
Add support for muted channels

### ISSUE THREAD

**Issue by decibyte** [erroneousboat/slack-term#229]
I've got a few channels muted in the official Slack app, because there's too much activity and I don't want to be notified all the time.

It would be nice to have support for this mute state, so muted channels will not get the "unread messages" indicator and not trigger the terminal bell (or whatever it's called).

**Comment by erroneousboat** [comment]
Yeah, seem reasonable. I'll have a look if that is supported in nlopes/slack
